docs/features.rst

```diff
@@ -23,66 +22,106 @@ For now, we extract the features for the whole recordings, and store them in sep
 We retrieve them by loading the whole feature matrix and selecting the relevant region (e.g. specified by a cut).
 Eventually we will look into optimizing the storage to further reduce the I/O.
 
-The feature matrices are accompanied by a YAML feature manifest. An example:
+There are two types of manifests:
```

Comment:
I notice above it says 'lilcom' is reducing the size by about half.  It takes a num_bits which defaults to -8, I think, which defines the 'tick size' or precision (-8 -> 1/256).  That may be a little too precise for log-energy features.   Perhaps -5 would be more appropriate as a default?

Reply:
Sure!